rocky/rocky/views/mixins.py

```diff
@@ -271,7 +270,7 @@ class HydratedReport:
     children_reports: list[Report] | None
     total_children_reports: int
     total_objects: int
-    report_type_summary: dict[str, int]
+    report_type_summary: dict[str, int] | None
```

Comment:
Why is None added?

Reply:
Some report types does not have a summary, I did a print of the log when accessing the summary:
<img width="1056" alt="image" src="https://github.com/minvws/nl-kat-coordination/assets/30832687/c21b4388-5567-485c-8034-21cc448017fc">

Reply:
it could return an empty set, should we use `Never` as typing I think

Reply:
The empty dict that it returns when there is no summary and `None` are two different things. Adding `None` because something that is returned can be empty is not correct. I'll remove the `None`.